src/ray/raylet/main.cc

```diff
@@ -47,6 +47,14 @@ int main(int argc, char *argv[]) {
   // Configuration for the object manager.
   ray::ObjectManagerConfig object_manager_config;
   object_manager_config.store_socket_name = store_socket_name;
+  // Time out in milliseconds to wait before retrying a failed pull.
+  object_manager_config.pull_timeout_ms = 100;
+  // Maximum number of sends allowed.
+  object_manager_config.max_sends = 2;
+  // Maximum number of receives allowed.
+  object_manager_config.max_receives = 2;
+  // Object chunk size, in bytes.
+  object_manager_config.object_chunk_size = static_cast<uint64_t>(std::pow(10, 8));
```

Comment:
I realize this has already been merged, but constants need to go in `ray_config.h`.